CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 1.5.0 - UNRELEASED
+### Notes
+The `logical_operators` rule is considered risky as operators `and` and `&&`
+have different precedence and in rare cases the logic of existing code could change.
+So to run this new version of the configuration, you need to adjust your composer scripts like this:
+
+```diff
+-	"cs:check": "php-cs-fixer fix --dry-run --diff",
++	"cs:check": "php-cs-fixer fix --dry-run --allow-risky yes --diff",
```

Comment:
it's a bit sad, that risky is required for checking as well…

Also you can also not run `cs:fix` or check skipping the risky, so the `cs:fix` also needs ` --allow-risky yes` or it can basically not be used anymore.

I assume everyone blindly accepts the `cs:fix` result anyway?
So we can also simply add it there.

It seems that the recommendation would be to configure the rule with:
```
		if ($this->getRiskyAllowed()) {
			$rules['logical_operators'] = true;
		}
```

In the Config we could default to allow risky then and apps can opt-out. So by default no further adjustments are required.